packages/circle-loader/stories/circleloader.stories.ts

```diff
@@ -19,73 +19,32 @@ export default {
 };
 
 export const Default = (): TemplateResult => {
-    const progress = number(
-        'Progress',
-        27,
-        {
-            range: true,
-            min: 0,
-            max: 100,
-            step: 1,
-        },
-        'Component'
-    );
-    const indeterminate = boolean('Indeterminate', false, 'Component');
     return html`
         <div
             style="width: 250px; height: 150px; display: flex; align-items: center; justify-content: space-around;"
         >
-            <sp-circle-loader
-                progress=${progress}
-                size="small"
-                ?indeterminate=${indeterminate}
-            ></sp-circle-loader>
-            <sp-circle-loader
-                progress=${progress}
-                ?indeterminate=${indeterminate}
-            ></sp-circle-loader>
-            <sp-circle-loader
-                progress=${progress}
-                size="large"
-                ?indeterminate=${indeterminate}
-            ></sp-circle-loader>
+            <sp-circle-loader progress="27" size="small"></sp-circle-loader>
+            <sp-circle-loader progress="27"></sp-circle-loader>
+            <sp-circle-loader progress="27" size="large"></sp-circle-loader>
         </div>
     `;
 };
 
 export const overBackground = (): TemplateResult => {
-    const progress = number(
-        'Progress',
-        53,
-        {
-            range: true,
-            min: 0,
-            max: 100,
-            step: 1,
-        },
-        'Component'
-    );
-    const indeterminate = boolean('Indeterminate', false, 'Component');
     return html`
         <div
             style="width: 250px; height: 150px; background-color: rgba(0,0,0,0.4); display: flex; align-items: center; justify-content: space-around;"
         >
             <sp-circle-loader
-                progress=${progress}
+                progress="53"
                 over-background
                 size="small"
-                ?indeterminate=${indeterminate}
-            ></sp-circle-loader>
-            <sp-circle-loader
-                progress=${progress}
-                over-background
-                ?indeterminate=${indeterminate}
             ></sp-circle-loader>
+            <sp-circle-loader progress="53" over-background></sp-circle-loader>
             <sp-circle-loader
-                progress=${progress}
+                progress="53"
                 over-background
                 size="large"
-                ?indeterminate=${indeterminate}
```

Comment:
The animation is hard to test, yes. We'd not been testing this specific story in the past (part of why there's the separate list 🙈), and this doesn't actually remove the capability from the story, it simply relying on the `withWebComponentsKnobs` helper which covers it dynamically rather than the manual application that was being used, as well, previously.